src/app/pages/SearchResultsPage/ResultsOnNetwork.tsx

```diff
@@ -15,23 +21,23 @@ import { SearchResults } from './hooks'
  */
 export const ResultsOnNetwork: FC<{
   network: Network
-  searchResults: SearchResults
+  searchQueries: SearchQueries
```

Comment:
First commit includes odd changes

Reply:
No longer. (I squashed them all 😁  )

(The separation of commits was no longer anyway, since the results of the first few commits were fully refactored later anyway.)

Reply:
:facepalm: no

Reply:
The first two commit used to be some code being renamed and being moved between files. But the state created there was also fully overhauled later, so the commits were no longer A->B  and then B->B',  but A->B and then B->C. Now, after squash, we have A->C instead. Since the concepts in version B didn't survive in the final version, I see no loss of information here. (And it was messed up because of too many rebasings anyway)